src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
I think we have this text in other places around the code. If that is correct, I'd use a similar wording (except if these have also "typos" in the text).